.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -37,6 +37,26 @@ You can find example PRs from this repository that include screenshots through t
 
 - [ ] Performance Analysis have been informed of the change
 
+## Acceptance tests have been updated
+
+- [ ] Any necessary changes to the [acceptance tests](https://github.com/govuk-one-login/authentication-acceptance-tests) have been made
```

Comment:
Are there any general kinds of changes that tend to break acceptance tests the most? I'm thinking like titles of pages, or changes to flow - maybe some other big stuff? Can we chuck them in?

Reply:
Good point - I'll check with the Test Engineers and update this to include them 👍🏽

Reply:
They've suggested changes to journeys. I had a look at the tests and thought back to things I've encountered so have also included changes to page titles and the text of interactive elements.